app/views/version-2/pre-htb/school-1/benefits-and-other-factors.html

```diff
@@ -0,0 +1,86 @@
+{% extends "version-2/layout.html" %}
+
+{% set pageTitle = "Benefits and other factors for " + projectData.name + " – Transfer an academy – GOV.UK" %}
+
+{% set projectData = data['records'][0] %}
```

Comment:
You'll want to swap the order of these two lines - files are read top to bottom, so if you want to refer to a variable (in this case `projectData`) you have to declare it first. So this should work if you swap the order :)

Reply:
Ah i see, that worked. Thank you @danielburnley